example-library/src/main/resources/application-example.yaml

```diff
@@ -16,7 +16,7 @@
 # Not interpreted by any java code, simply used by reference in the population.resources section
 default:
   rootId: ../Palisade-examples/resources
-  connectionDetail: data-service
+  connectionDetail: localhost:8082
```

Comment:
Just a question.  We use data-service in other aspects of the code, but here it is changed a port on localhost.  Why is this different?

Reply:
In the application-example we are running a static example on localhost and multiple JVMs, so this specifies where the client should expect the data-service to be located.